content/technical-guide/installation/supported-environments.md

```diff
@@ -21,9 +21,9 @@ The following environments are supported:
 
 # Web Browser
 
-- Google Chrome latest (currently v123.0) [recommended]
-- Mozilla Firefox latest (currently v124.0)
-- Microsoft Edge latest (currently v123.0)
+- Google Chrome latest (currently v124.0) [recommended]
+- Mozilla Firefox latest (currently v125.0.1)
```

Comment:
🙃 I think you don't need to provide the patch level here:
```suggestion
- Mozilla Firefox latest (currently v125.0)
```

Reply:
Okay, I looked at the [release notes](https://www.mozilla.org/en-US/firefox/125.0.1/releasenotes/), which stated: "NOTE: Due to a high-severity quality issue discovered shortly before release, the 125.0 release was skipped in favor of 125.0.1.". But it makes sense to not providing patch level here.